infra/ci.rkt

```diff
@@ -41,10 +41,17 @@
     (match status
       ['success
        (match-define 
-        (improve-result preprocess pctxs start target end bogosity) backend)
+        (improve-result preprocess pctxs start targets end bogosity) backend)
        (match-define (alt-analysis start-alt _ start-error) start)
        (match-define (alt-analysis end-alt _ end-error) (first end))
-       (define target-error (and target (alt-analysis-test-errors target)))
+
+      ;; Pick lowest target from all target
+      (define target-error
+        (cond
+          [(empty? targets) #f] ; If the list is empty, return false
+          [else
+            (alt-analysis-test-errors 
+              (argmin (lambda (target) (errors-score (alt-analysis-test-errors target))) targets))]))
```

Comment:
```suggestion
            (argmin errors-score (map alt-analysis-test-errors targets))]))
```

Reply:
Also instead of a `cond` why not just do `(and (not (empty? targets)) ...)`

Reply:
Or just an `if` statement?

Reply:
Done. Initially I thought there would be more cases